### PR TITLE
Translate 2.5.1 release post (RU)

### PR DESCRIPTION
**Pull request by arbox**
I've translated the Release Notice for Ruby 2.5.1.

Please review: @gazay , @razum2um

### CODE REVIEW

File: ru/news/_posts/2018-03-28-ruby-2-5-1-released.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+title: "Вышел Ruby 2.5.1"
+author: "naruse"
+translator: "arbox"
+date: 2018-03-28 17:20:00 +0000
+lang: ru
+---
+
+Вышел Ruby 2.5.1.
+
+Этот релиз влючает в себя исправление ошибок и решений проблем в безопасности.
```
Comment (razum2um) [inline ru/news/_posts/2018-03-28-ruby-2-5-1-released.md]: (влючает) решения

File: ru/news/_posts/2018-03-28-ruby-2-5-1-released.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+* [CVE-2017-17742: HTTP response splitting in WEBrick](/en/news/2018/03/28/http-response-splitting-in-webrick-cve-2017-17742/)
```
Comment (razum2um) [inline ru/news/_posts/2018-03-28-ruby-2-5-1-released.md]: @arbox let's translate descriptions?

Comment (arbox) [reply ru/news/_posts/2018-03-28-ruby-2-5-1-released.md]: @razum2um we definitely need to...
I'd like to translate all related security releases and update links to them.

**Comment by lex111** [comment]
@arbox Thanks!

**Pull request merged**